src/test/java/FIleUtils/CSVReaderTest.java

```diff
@@ -0,0 +1,33 @@
+package FIleUtils;
+
+import FileUtils.CSVReader;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+
+/**
+ * Created by p998tbd on 2016.12.13.
+ */
+public class CSVReaderTest {
+
+    private File file;
+
+    @Before
+    public void setUp() {
+        ClassLoader classLoader = getClass().getClassLoader();
+        file  = new File(classLoader.getResource("testData.txt").getFile());
+    }
+
+    @Test
+    public void loadFileTest() throws IOException {
+        List<List<String>> result = new CSVReader().loadFile(file.getAbsolutePath().toString());
+        Assert.assertFalse(result.isEmpty());
+        Assert.assertTrue(result.size() == 5);
```

Comment:
Maybe it would be more readable in this way:
`Assert.assertEquals(5, result.size());`